src/core/ContainerChildren/index.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+export const ContainerChildren = styled.div`
+  background-color: #cfd0d190;
+  border-radius: 5px;
+  padding: 10px;
+  //box-shadow: 5px 2px 20px 1px rgba(138, 136, 136, 0.75);
```

Comment:
```suggestion
```

It's better to remove unused code, you can always find it again in the git history if you need it